dependencies.yaml

```diff
@@ -324,13 +324,13 @@ dependencies:
       match: \d+.\d+(alpha|beta|rc)?\.?(\d+)?
 
   - name: "golang: after kubernetes/kubernetes update (previous release branches: master)"
-    version: 1.21.2
+    version: 1.21.3
     refPaths:
     - path: images/releng/k8s-ci-builder/variants.yaml
       match: \d+.\d+(alpha|beta|rc)?\.?(\d+)?
 
   - name: "golang: after kubernetes/kubernetes update (previous release branches: 1.28)"
-    version: 1.20.9
+    version: 1.20.10
```

Comment:
Did the other prs for the active branches get merged?

Reply:
No, there's some issue with tests on all release branches

Reply:
I will take a look tomorrow on those :/